views/dpirate.ejs

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```

Comment:
Remember, you can use partials to DRY up your code so you don't have to write the same code for views over and over again!